aws/lambda/opensearch-gha-jobs/Makefile

```diff
@@ -0,0 +1,11 @@
+prepare: clean
+	mkdir -p ./packages
+	pip install --target ./packages -r requirements.txt
```

Comment:
pip vs pip3? Is this expecting to be run from inside a virtual environment or something?

Might be helpful to add some comments explaining how this makefile should be invoked

Reply:
Ok, let me add a README, `make deploy` is the command to use to deploy the new version of the lambda.  I think I will also add a CI for this to run the test.